app-misc/fff/fff-0.3.2.recipe

```diff
@@ -0,0 +1,39 @@
+SUMMARY="A simple file manager written in bash"
+DESCRIPTION="Features:
+* Fast
+* Minimal (~100 lines of bash)
+* Smooth Scrolling (using vim keybindings)
+* File Oprations (copy, paste, rename, cut, etc)
+* Instant as you type search"
+HOMEPAGE="https://github.com/dylanaraps/fff"
+COPYRIGHT="2016-2018 Dylan Araps"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="f4d64abd0adacfdcc3d2e8f8adc80a878c41b982c662a97e8b4f1989f01aea61"
+SOURCE_FILENAME="$portVersionedName.tar.gz"
+PATCHES="fff-0.3.2.patch"
+
+ARCHITECTURES="any"
+
+GLOBAL_WRITABLE_FILES="
+	cache/fff/trash directory
+	"
+
+PROVIDES="
+	fff = $portVersion
+	cmd:fff
+	"
+REQUIRES="
+	bash
+	cmd:file
+	haiku
```

Comment:
The `*REQUIRES` variable have an "unofficial" sorting convention: `haiku` is always the first entry, after that the package names, then `add_on:`, `app:`, `cmd:` and `lib:`.